src/dos/program_tree.cpp

```diff
@@ -424,12 +424,12 @@ void TREE::AddMessages()
 	        "            with an optional [color=white]-[reset] prefix to reverse order.\n"
 	        "  /a        uses only 7-bit ASCII characters.\n"
 	        "  /b        brief display, omits header and footer information.\n"
-	        "  /f        also display files.\n"
-	        "  /p        display one page a time, shorten output to fit width width.\n"
-	        "  /da       display attributes.\n"
-	        "  /df       display size for files.\n"
-	        "  /dh       also display hidden and system files/directories.\n"
-	        "  /o[color=white]ORDER[reset]   orders the list (see above)\n"
+	        "  /f        also displays files.\n"
+	        "  /p        displays one page a time, shorten output to fit screen width.\n"
+	        "  /da       displays attributes.\n"
+	        "  /df       displays size for files.\n"
+	        "  /dh       also displays hidden and system files/directories.\n"
```

Comment:
"Display" was fine for all of these; it's the "command form" or "imperative mood" (I'm not a grammar expert 😄). Can you change these back please, and rather fix up the ones you haven't touched? (e.g. "use only 7-bit" instead of "uses", "omit header" instead of "omits" etc.)

Generally, we want "imperative mood" for command line option descripitons.

Reply:
I changed it to "displays" because it is part of the "Where:" section, so I thought it should be read "Where /p displays...". But I can change it everywhere if you want.

Regarding the first line of help messages, it is like that for every command (checking "help /a" I've just noticed that only the "more" description is different...). And it's also like that on MS-DOS and FreeDOS. Should I change it for every command anyways?

Reply:
Yeah, it should be like the `MORE` command, so imperative everywhere (e.g., "Copy one or more files", not "Copies"). If you have the inclination to change this for all commands, please go for it 😄